packages/server/src/constants/index.ts

```diff
@@ -87,12 +95,10 @@ export const USERS_TABLE_SCHEMA = {
         },
         presence: true,
       },
-      fieldName: "email",
```

Comment:
How come this `fieldName` are dropped?

Reply:
it really shouldn't there - this wasn't typed previously and the `fieldName` only pertains to relationships. Won't cause any problems - but no need for it to be there.